src/components/error-info.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Collapsible from 'react-collapsible';
+
+export default function ErrorInfo({ error, engineURL }) {
+  let showForm = true;
+  let showMoreInfo = true;
+
+  const learnMore = (
+    <Collapsible trigger="Learn more">
```

Comment:
Maybe some other name for this but I can't come up with a better phrase

Reply:
Not new for this PR. But I kind of agree.. if you come up with a better phrase, create an issue!